src/backend/joanie/core/certifications/models.py

```diff
@@ -0,0 +1,74 @@
+import uuid
```

Comment:
Ab out the way to structure the code, I would like to discuss putting everything in one app. If there are too many models, we can make `models` a directory and have several files inside this directory.  But I feel there are no reasons yet to split to separate apps. This is the philosophy we followed in https://github.com/openfun/richie and https://github.com/openfun/marsha. In richie there is a `demo` and a `search` app because they are optional features added to the core app. In you case, everything is intertwined and your apps require each other... so imo all this should be in a single app.